src/theory/quantifiers/sygus/cegis_core_connective.cpp

```diff
@@ -379,7 +381,7 @@ Node CegisCoreConnective::Component::getSygusSolution(
   std::sort(conjs.begin(), conjs.end());
   Node sol;
   std::map<Node, Node>::const_iterator itu;
-  NodeManager* nm = NodeManager::currentNM();
+  NodeManager* nm = d_nm;
```

Comment:
Same as above, why not use `d_nm` directly?

Reply:
This case is different. `CegisCoreConnective::Component` has a `d_nm` member. Changed.

Reply:
After merging PR #11403 , the member `d_nm` is no longer necessary.